helm/oauth2-proxy/templates/configmap.yaml

```diff
@@ -13,6 +13,13 @@ metadata:
   name: {{ template "oauth2-proxy.fullname" . }}
   namespace: {{ template "oauth2-proxy.namespace" $ }}
 data:
+  {{- if and (not .Values.alphaConfig.enabled) .Values.config.legacyDefaults }}
+  oauth2_proxy.cfg: |-
+{{ tpl .Values.config.legacyDefaults $ | indent 4 }}
+
+{{ tpl .Values.config.configFile $ | indent 4 }}
```

Comment:
This will actually cause a breaking change for existing deployments:

```
[2025/04/30 18:30:31] [main.go:42] ERROR: failed to load config: unable to load config file: While parsing config: toml: key upstreams is already defined
```